test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -45,22 +45,58 @@ public async Task GivenPatients_WhenSearchedWithSortParamsWithHyphen_ThenPatient
             await ExecuteAndValidateBundle($"Patient?_tag={tag}&_sort=-_lastUpdated", false, patients.Reverse().Cast<Resource>().ToArray());
         }
 
-        [Fact]
-        public async Task GivenMoreThanTenPatients_WhenSearchedWithSortParam_ThenPatientsAreReturnedInAscendingOrder()
+        [Theory]
```

Comment:
These should be [SkippableTheory] because birthdate might not be enabled on all dbs

Reply:
Sent PR #1864  to address this